amps/ags/rm-automation/rm-automation-community-rest-api/src/main/java/org/alfresco/rest/rm/community/model/hold/HoldBulkStatus.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * #%L
+ * Alfresco Records Management Module
+ * %%
+ * Copyright (C) 2005 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * -
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ * -
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * -
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * -
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.rest.rm.community.model.hold;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.alfresco.utility.model.TestModel;
+
+/**
+ * POJO for hold bulk request
+ *
+ * @author Damian Ujma
+ */
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class HoldBulkStatus extends TestModel
+{
+    @JsonProperty
+    private String bulkStatusId;
+
+    @JsonProperty
+    private String startTime;
+
+    @JsonProperty
+    private String endTime;
+
+    @JsonProperty
+    private long processedItems;
+
+    @JsonProperty
+    private long errorsCount;
+
+    @JsonProperty
+    private long totalItems;
+
+    @JsonProperty
+    private String lastError;
+
+    @JsonProperty
+    private Status status;
+
+    public enum Status
+    {
+        PENDING,
+        IN_PROGRESS,
+        DONE
+    }
+
+    @Override
+    public boolean equals(Object o)
```

Comment:
You already defined `equals` and `hashCode` with `@Data`

Reply:
Removed.